docs/3. Product Features/Interactive Messages Re-response.md

```
@@ -0,0 +1,44 @@
+> ### **2 minute read &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; `Beginner`**
+
+## Handling re-responses on the interactive messages
+
+**This is a flagged feature, and can be enabled for your organization’s glific instance upon request to Glific team. It is advisable to read up on how this feature works and take a decision based on your organization’s use case. This feature can be enabled for the organization upon request. Reach out to support@glific.org or on Discord NGO support channel with a message to “Enable the feature to handle interactive message re-response**
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)